### ISSUE TITLE
Flashing Receiver Firmware via External Bay fails (comms error).

### ISSUE THREAD

**Issue by Lex-Dysia** [EdgeTX/edgetx.github.io#62]
Hello, I'm running V2.7.1 on a Radiomaster TX16S. Today for the first time (on EdgeTX) I tried to flash a RadioMaster R168 receiver via the external bay of the radio. I tried 2 different versions of the firmware (.frk files) several times all resulting in failure messages. Protocol was set to PPM.

I then tried the same exercise with the same firmware files and the same receiver on a Taranis running a recent version of OpenTX and everything worked correctly.

**Comment by Lex-Dysia** [comment]
I'm wondering if I've logged this fault to the wrong location. No response or assignment of problem to date??
Should I be flagging this somewhere else?
Thanks.

**Comment by phileville** [comment]
You are correct, this repository is for the EdgeTX Webpage.  I will move it to the main EdgeTX Repo.
